docs/cases/cases-overview.asciidoc

```diff
@@ -118,9 +118,9 @@ IMPORTANT: Before importing Lens visualizations, Timelines, or alerts into a spa
 [float]
 [[cases-export]]
 === Export a case
-Use the export option to move cases between different Kibana instances. When you export a case, the following data is exported to a newline-delimited JSON (NDJSON) file: case details, user actions, text string comments, case alerts, and lens visualizations.
+Use the export option to move cases between different Kibana instances. When you export a case, the following data is exported to a newline-delimited JSON (`.ndjson`) file: case details, user actions, text string comments, case alerts, and lens visualizations (which are exported as JSON blobs).
 
-NOTE: Exported alerts contain metadata that allow the case to reference the appropriate alerts after they're imported. Lens visualizations are exported as JSON blobs.
+IMPORTANT: Alerts attached to a case are exported as references containing alert metadata. When a <<cases-import, case is imported>>, it uses these references to re-connect to the appropriate alerts. However, alert references only resolve if the correct alert documents are available in the environment. If they're not, alert details will not load when you access them from the imported case. 
```

Comment:
Thank you @nastasha-solomon! This is much better. Maybe we can add at the end "The same applies to the rules from which the alerts have been created" or similar 🙂